src/main/java/org/jabref/gui/maintable/MainTableColumnModel.java

```diff
@@ -36,6 +36,7 @@ public class MainTableColumnModel {
     private static final Logger LOGGER = LoggerFactory.getLogger(MainTableColumnModel.class);
     public enum Type {
         MATCH_CATEGORY("match_category", Localization.lang("Match category")),
+        SCORE("search_score", Localization.lang("Search score")),
```

Comment:
Parallel to the float search PR one could argue, that this should also be called MATCH_SCORE ore something likewise... @koppor

Reply:
Done in beb3881183